toolkit/tools/imagegen/attendedinstaller/views/installerview/installerview.go

```diff
@@ -63,8 +63,22 @@ func New(calamaresInstallFunc func()) *InstallerView {
 	if err != nil {
 		logger.Log.Debugf("Calamares not found, defaulting to terminal based installer")
 	} else {
-		logger.Log.Debugf("Calamares found, but the gui installer is temporarily disabled until stabilized")
-		// iv.installerOptions = append(iv.installerOptions, uitext.InstallerGraphicalOption)
+		//logger.Log.Debugf("Calamares found, but the gui installer is temporarily disabled until stabilized")
+		iv.installerOptions = append(iv.installerOptions, uitext.InstallerGraphicalOption)
+
+		logger.Log.Debugf("Running chgrp for dbus-daemon-launch-helper")
+		cmd := exec.Command("chgrp", "messagebus", "/usr/libexec/dbus-daemon-launch-helper")
+		err = cmd.Run()
+		if err != nil {
+			logger.Log.Debugf("Error while running chgrp for dbus-daemon-launch-helper")
+		}
+		logger.Log.Debugf("Running chmod for dbus-daemon-launch-helper")
+		cmd = exec.Command("chmod", "4750", "/usr/libexec/dbus-daemon-launch-helper")
+		err = cmd.Run()
+		if err != nil {
+			logger.Log.Debugf("Error while running chmod for dbus-daemon-launch-helper")
+		}
```

Comment:
Why do we manually change the group and permissions for this file from our tooling? I understood you've hit issue with `dbus-daemon-launch-helper` but this looks hacky.

Reply:
This change is required within the initrd environment and was the cleanest way I could easily come up with in order to achieve the result. Adding the additional permissions within `dbus.spec` was not a sufficient fix as during chroot creation all files are assigned the to user and group "root". Even trying to manually set them during chroot creation from within the chroot environment was not successful. After such unsuccessful, cleaner fixes, this is what I was left with as the option to keep things moving (short of shunting this permissions check/fix off to the calamares code somewhere via a patch).